Softeq.XToolkit.Common.Tests/Tasks/TaskDeferralTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Threading.Tasks;
+using Softeq.XToolkit.Common.Tasks;
+using Xunit;
+
+namespace Softeq.XToolkit.Common.Tests.Tasks
+{
+    public class TaskDeferralTests
+    {
+        private readonly TaskDeferral<string> _deferral;
+
+        public TaskDeferralTests()
+        {
+            _deferral = new TaskDeferral<string>();
```

Comment:
You can actually get rid of the constructor and just use `private readonly TaskDeferral<string> _deferral = TaskDeferral<string>();`